modules/misc/news.nix

```diff
@@ -1891,6 +1891,14 @@ in
         '';
       }
 
+      {
+        time = "2021-04-04T10:00:00+00:00";
+        condition = hostPlatform.isLinux;
+        message = ''
+          A new module is available: 'services.pass-secret-service'.
+        '';
+      }
+
```

Comment:
I think this should be appended to the list (i.e. last) and the time updated to the current date.
I don't know how HM generates/reads the news file entry.